RubberduckTests/Inspections/EmptyMethodInspectionTests.cs

```diff
@@ -94,7 +71,7 @@ public void EmptyMethod_ReturnsExpectedMessage()
                 var inspector = InspectionsHelper.GetInspector(inspection);
                 var actualResults = inspector.FindIssuesAsync(state, CancellationToken.None).Result;
 
-                Assert.AreEqual("Property Get 'Bar' contains no executable statements.", actualResults.Select(result => result.Description).First());
+                Assert.AreEqual("Property get accessor 'Bar' contains no executable statements.", actualResults.Select(result => result.Description).First());
```

Comment:
I think this test is dependent on locale, I'd expect it to fail in e.g. a German environment. It will also break if the wording of the resource is later rephrased, which makes it more brittle than it needs to be - I'd just remove this test: there's already one that will fail the build if any inspection is missing any of the resource strings.